week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "importHelpers": true,
+    "alwaysStrict": true,
+    "sourceMap": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "baseUrl": "./src",
+    "paths": {
+      "framework": [
+        "framework"
+      ],
+      "shop": [
+        "shop"
+      ],
+      "types": [
+        "types"
+      ]
+    }
+  },
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist"
+  ]
+}
```

Comment:
Nie generowałeś pliku za pomocą `tsc --init` ? 
/ to nic złego, ale /
Kojarzysz, że tak też można?

Reply:
Generowałem plik za pomocą `tsc --init`, ale go wyczyściłem z komentarzy i dodałem kilka rzeczy 🤔